### ISSUE TITLE
DOT writer

### ISSUE THREAD

**Issue by davidalber** [davidalber/geneagrapher#6]
Create a writer class that generates [DOT](http://en.wikipedia.org/wiki/DOT_language) output for the graph.

---

Blocked by #5.

**Comment by davidalber** [comment]
This issue is no longer relevant due to the changes made in [v2.0.0](https://github.com/davidalber/geneagrapher/releases/tag/v2.0.0).
